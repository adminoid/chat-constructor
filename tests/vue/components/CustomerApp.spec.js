const tu = require('@vue/test-utils');
const CustomerApp = require('@customer/ts/components/CustomerApp.vue').default;

describe('CustomerApp', () => {

  const wrapper = tu.shallowMount(CustomerApp);

  it('should contain Sidebar', () => {
    expect(wrapper.text().includes('Sidebar')).toBe(true);
  });

  it('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBe(true)
  });

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })

});
