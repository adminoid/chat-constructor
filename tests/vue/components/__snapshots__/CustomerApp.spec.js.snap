// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomerApp renders correctly 1`] = `
<div
  class="row no-gutters"
>
  <aside
    class="sidebar col-md-3"
  >
    <div
      class="sidebar__area"
    >
      Sidebar
    </div>
  </aside>
  <section
    class="col-md-9"
  >
    <div
      class="top-panel"
    >
      <button
        class="top-panel__add-block-btn btn btn-success"
        type="button"
      >
        Добавить блок
      </button>
    </div>
    <section
      class="main-container"
    />
  </section>
</div>
`;
